pkg/systemd/host.js

```diff
@@ -334,10 +335,40 @@ PageServer.prototype = {
         var pmcd_service = service.proxy("pmcd");
         var pmlogger_service = service.proxy("pmlogger");
         var pmlogger_promise;
+        var pmlogger_exists = false;
+
+        function refresh_pmlogger_state() {
+            /* HACK: The pcp packages on Ubuntu and Debian inclue SysV init
+             * scripts in /etc, which stay around when removing (as opposed to
+             * purging) the package. Systemd treats those as valid units, even
+             * if they're not backed by packages anymore.  Thus, and
+             * pmlogger_service.exists will be true.  Check for the binary
+             * directly so that we don't show a malfunctioning switch in this
+             * case, but the offer to install PCP.
+             */
+            cockpit.spawn([ "which", "pmlogger" ], { err: "ignore" })
+                .then(function () {
+                    pmlogger_exists = true;
+                })
+                .fail(function () {
+                    pmlogger_exists = false;
+                })
+                .always(function () {
+                    if (!pmlogger_exists) {
+                        $('#system-information-enable-pcp').show();
+                        $('#server-pmlogger-onoff-row').hide();
+                    } else if (!pmlogger_promise) {
+                        $('#system-information-enable-pcp').hide();
+                        console.log(pmlogger_service);
```

Comment:
Does this even show something sensible? More than just `[Object]`?

Reply:
It doesn't. I used it while debugging an forgot to take it out.